### PR TITLE
Add intermediate tablet breakpoint to improve view on small screens

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Added new screen width for correct display of forms on small screens (no empty space on the right) & Reduced width of the tree and main menu.

And reduced the width of tree and menu on the left by a few pixels.

Before:
width_1024

**After:**
width_size

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/15247

### CODE REVIEW

**Comment by Ruslan-Aleev** [comment]
By the way, maybe it makes sense to change the size of the grids to a more standardized one (bootstrap, for example)? I'm about:
https://github.com/modxcms/revolution/pull/15259/files#diff-8f75bc91b58ef5d65f0f46f1268c1250L222

File: _build/templates/default/sass/_colors-and-vars.scss [hunk 1]
```diff
@@ -222,12 +222,14 @@ $mobile:  (media: "screen and (max-width: 640px)", columns: 4);
 $tabletP: (media: "screen and (max-width: 768px)", columns: 12);
 $desktop: (media: "screen and (max-width: 960px)", columns: 12);
 $tabletL: (media: "screen and (max-width: 1024px)", columns: 12);
+$tabletM: (media: "screen and (max-width: 1140px)", columns: 12);
```
Comment (mishanthrop) [inline _build/templates/default/sass/_colors-and-vars.scss]: Why `M` is larger than `L`, if `M` for `Medium` and `L` for `Large`?

File: _build/templates/default/sass/_navbar.scss [hunk 2]
```diff
@@ -38,7 +38,7 @@
   z-index: 20;
   display: flex;
   flex-direction: column;
-  padding: 0 10px;
+  padding: 0 5px;
```
Comment (mishanthrop) [inline _build/templates/default/sass/_navbar.scss:41]: Is this PR for adding new screen width or for changing current? Why do you change default `#modx-navbar padding`, `.x-tab-panel-noborder margin` etc?

**Review by Ibochkarev — APPROVED** [review]

**Comment by Ruslan-Aleev** [comment]
@opengeek PRs do not pass checks, but there are no visual errors.

File: _build/templates/default/sass/_navbar.scss (hunk 2 above)
Comment (JoshuaLuckers) [reply _build/templates/default/sass/_navbar.scss:41]: @Ruslan-Aleev can you answer @mishantrop's question? :D

Comment (Ruslan-Aleev) [reply _build/templates/default/sass/_navbar.scss:41]: I reduced the width of menu and tree by a few pixels, it is clear from commits.
Changed the title to make it clearer.

Comment (Jako) [reply _build/templates/default/sass/_navbar.scss:41]: But the PR title speaks of small screens and the changes are mostly for all widths. This is somehow misleading.

**Review by alroniks — APPROVED** [review]

**Pull request merged**